packages/components/src/organisms/RatingSummary/RatingSummary.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { forwardRef, type HTMLAttributes } from 'react'
+import Rating from '../../molecules/Rating'
+import ProgressStatus from '../../molecules/ProgressStatus'
+import Icon from '../../atoms/Icon'
+import Button from '../../atoms/Button'
+
+export interface RatingSummaryProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * The average rating of the product
+   */
+  average: number
+  /**
+   * The total number of reviews of the product
+   */
+  totalCount: number
+  /**
+   * The distribution of the ratings
+   */
+  distribution: {
+    starsOne: number
+    starsTwo: number
+    starsThree: number
+    starsFour: number
+    starsFive: number
+  }
+  /**
+   * The total count text to be displayed after the total count number
+   * @default 'Reviews'
+   */
+  totalCountText?: string
+  /**
+   * The text to be displayed when there are no reviews
+   * @default 'No reviews yet'
+   */
+  zeroCountText?: string
+  /**
+   * The text to be displayed on the button for writing a new review
+   * @default 'Write a review'
+   */
+  buttonText?: string
+  /**
+   * The text to be displayed on the button for writing the very first review
+   * @default 'Write the first review'
+   */
+  buttonTextFirstReview?: string
+  /**
+   * Optional test ID for testing.
+   */
+  testId?: string
+}
+
+function buildProgressStatus(key: number, value: number) {
+  return (
+    <ProgressStatus
+      data-fs-rating-summary-distribution-status
+      leftSideElement={
+        <span data-fs-rating-summary-distribution-key>
+          <p>{key}</p>
```

Comment:
Add a `width: var(--fs-spacing-1);` on this so all the numbers and starts are aligned vertically.